air-quality-ui/src/services/forecast-data-service.ts

```diff
@@ -9,14 +9,23 @@ export const getForecastData = async (
   dateFrom: DateTime,
   dateTo: DateTime,
   baseTime: DateTime,
+  location_name?: string,
   locationType: LocationType = 'city',
 ): Promise<ForecastResponseDto[]> => {
-  const params: Record<string, string> = {
-    location_type: locationType,
-    valid_time_from: dateFrom.toJSDate().toISOString(),
-    valid_time_to: dateTo.toJSDate().toISOString(),
-    base_time: baseTime.toJSDate().toISOString(),
-  }
+  const params: Record<string, string> = location_name
```

Comment:
This isn't very extensible and duplicates code unnecessarily. Can either just add an if statement after the original object and append location_name to it or this would work:

```
{
        location_type: locationType,
        valid_time_from: dateFrom.toJSDate().toISOString(),
        valid_time_to: dateTo.toJSDate().toISOString(),
        base_time: baseTime.toJSDate().toISOString(),
        ...(location_name && { location_name: locationName }),
      }
```